include/rom.h

```diff
@@ -3,82 +3,96 @@
 
 #include "xlFileGCN.h"
 
+typedef s32 UnknownCallbackFunc();
```

Comment:
As discussed, because C is weird this should probably be

```suggestion
typedef s32 UnknownCallbackFunc(void);
```

same with __romCopyUpdate_Complete() and __romCopyUpdate_Complete()